features/pages/flaw_detail_page.py

```diff
@@ -9,7 +9,13 @@ def __init__(self, driver):
 
     locators = {
         "addCommentBtn": ("XPATH", "//button[contains(text(), 'Comment')]"),
-        "commentTextWindow": ("XPATH", "(//textarea[@class='form-control'])[5]")
+        "commentTextWindow": ("XPATH", "(//textarea[@class='form-control col-9 d-inline-block'])[5]"),
+        "statementText": ("XPATH", "//span[text()='Statement']"),
+        "statementTextWindow": ("XPATH", "(//textarea[@class='form-control col-9 d-inline-block'])[3]"),
+        "saveBtn": ("XPATH", '//button[text()="Save Changes"]'),
+        "flawSavedMsg": ("XPATH", "//div[text()='Flaw saved']"),
+        "mitigationText": ("XPATH", "//span[text()='Mitigation']"),
```

Comment:
It seems this locator won't be used.

Reply:
Yes. It could be used in other case. I think we can keep it.